### ISSUE TITLE
Maximum call frequency setFundingRate

### ISSUE THREAD

**Issue by Waltermelon-lang** [nexlabs22/NEX-Dex-contracts#90]
function setFundingRate() public onlyOwner {

-   Make sure this one can only be called say every 55 minutes, good idea?
-   Instead of calculating, fetch uint256 fundingFee from our private oracle.

**Comment by Morteza3123** [comment]
Very good idea. Do you mean to add require to not able to call lower than 55 minutes after the latest call time? (if yes it's ok for me to implement this limitation)
And another question is that do you want this 55 minutes for calling funding rate from oracle?

**Comment by Waltermelon-lang** [comment]
Very good idea. Do you mean to add require to not able to call lower than 55 minutes after the latest call time? (if yes it's ok for me to implement this limitation)
I think yes. 
And another question is that do you want this 55 minutes for calling funding rate from oracle?
what do you mean form oracle:? we only call the funding rate from our oracle. I suppose the answer is yes, but I don't follow
the reasoning behind this question.

**Comment by Morteza3123** [comment]
Yes now we have our own oracle so I mean our oracle. It's ok I share the code after adding this limitation.
